central/probeupload/manager/manager_impl.go

```diff
@@ -219,114 +157,56 @@ func (m *manager) StoreFile(ctx context.Context, file string, data io.Reader, si
 		return errors.Errorf("invalid file name %q", file)
 	}
 
-	if freeBytes, err := fsutils.AvailableBytesIn(m.rootDir); err == nil {
-		if freeBytes-uint64(size)-uint64(fileSizeOverhead) < uint64(m.freeDiskThreshold) {
-			return errors.Errorf("only %d bytes left on partition, not storing probes to avoid impacting database health", freeBytes)
+	// When using managed services, Postgres space is not a concern.
```

Comment:
Can you add a TODO for ROX-16407?  We won't be able to determine this in BYODB circumstances either.